protocol-specs/generators/multisignature_registration/index.js

```diff
@@ -139,17 +143,30 @@ const serializeMemberSignatures = (tx, txBuffer) => {
 
 	tx.signatures.forEach(aSignature => {
 		const signatureBuffer = Buffer.concat([hexToBuffer(aSignature)]);
-		txBufferCopy = Buffer.concat([txBufferCopy, signatureBuffer]);
+		if (signatureBuffer.lenght === 0) {
+			txBufferCopy = Buffer.concat([
+				txBufferCopy,
```

Comment:
Does this really work? I mean having `txBufferCopy` on the left and the right side of the assignment without having `txBufferCopy` initialized before?

Reply:
it's initialized above in line 141 :) 

```	
let txBufferCopy = Buffer.alloc(txBuffer.length);
txBuffer.copy(txBufferCopy);
```

Reply:
ah, thanks. I overlooked it during the review :)